docs/user/troubleshooting.md

```diff
@@ -89,3 +89,59 @@ This is safe to ignore and merely indicates that the etcd bootstrapping is still
 ### Installer Fails to Create Resources
 
 The easiest way to get more debugging information from the installer is to increase the logging level. This can be done by adding `--log-level=debug` to the command line arguments. Of course, this cannot be retroactively applied, so it won't help to debug an installation that has already failed. The installation will have to be attempted again.
+
+## Generic Troubleshooting
+
+Here are some ideas if none of the [common failures](#common-failures) match your symptoms.
+For other generic troubleshooting, see [the Kubernetes documentation][kubernetes-debug].
+
+### Check for Pending or Crashing Pods
+
+This is the generic version of the [*No Worker Nodes Created*](#no-worker-nodes-created) troubleshooting procedure.
```

Comment:
this is already part of `## Generic Troubleshooting` why explain again?

Reply:
> this is already part of `## Generic Troubleshooting` why explain again?

I wanted to link back to the *No Worker Nodes Created* section.  Can you float alternative phrasing for that link?